addon/templates/components/flexberry-maplayer.hbs

```diff
@@ -41,6 +43,12 @@
       <div style="clear: both;"></div>
     {{/if}}
 
+    {{#if _canChangeOpacity}}
+      <label class="{{flexberryClassNames.opacityLabel}}" title="{{t "components.flexberry-maplayer.opacity" }}">
```

Comment:
It was Lena's task to make opacity value always visible for user, just like on Rosreestr's cadastral map. Use case of that is following: user can set opacity to 0 and then just forget about that, and then he will be shocked 'Why that layer is not displaying??', some kind of that.
preventExpandCollapse wasn't applied for next reason: when user see the value of opacity and wants to change it, he often automatically clicks on that label or whatever else (we think so).  Dropdown's arrow is not obvious button to do that.
Basically, it was considered to use semantic UI popup and do not place slider in accordion's content, but the situation was the same as 'accordion inside accordion', it didn't work.
Possibly, it is some kind of idea to make it just 'label', but with the same behavior as now.